docs/welcome.md

```diff
@@ -1,52 +1,51 @@
 ---
 id: welcome
-title: Getting Started
+title: Get Started
 description: Master Amplication with in-depth developer guides, API details, and tutorials.
 sidebar_label: Introduction
 slug: /
 ---
 
 # Get Started with Amplication
 
-Hello and welcome to Amplication's developer documentation.
+Welcome to Amplication's developer documentation!
 
-[Amplication](/about/) is an open-source backend development tool. It accelerates the development of Node.js and .NET applications by auto-generating production-ready backends.
-
-Streamline your development process and increase productivity so you can focus on what matters most: **creating awesome applications**.
+Amplication is an open-source, AI-powered backend development platform that accelerates the creation of production-ready Node.js and .NET applications. With Amplication, you can focus on your core business logic while we handle the boilerplate.
 
 ## Quick Start
 
-[Create your first service](/first-service/) with Amplication.
-Set up a new service in less than 1 minute with our onboarding wizard.
-
-## Key Features of Amplication
-
-Amplication helps you build a fully functional backend in minutes. Just define your [data models](/how-to/create-entity/), roles, and [permissions](/how-to/set-access-permissions/) — and Amplication will generate a Node.js or .NET application with everything you need.
-
-All generated applications include the following features:
+Get up and running in minutes:
```

Comment:
Let's mention here (again)- production-ready / persistent / consistent etc. + AI capabilities

Reply:
Let's also mention (again) .NET and Node.js